config/constants.go

```diff
@@ -110,4 +110,7 @@ const (
 	// WorkspaceVolumeName contains the name of the volume that is shared between
 	// the init containers and containers for a driver or worker pod.
 	WorkspaceVolumeName = "workspace"
+
+	// KillAfter sets the allowed response time for a pod after timeout.
+	KillAfterSeconds = "KILL_AFTER_SECONDS"
```

Comment:
I think we want "KILL_AFTER" here.

Reply:
I forgot the modify KILL_AFTER to KILL_AFTER_SECONDS earlier in the run.sh file, sorry for the confusion. I think it makes scene to me o be consistent.